content/doc/book/installing/installing-on-different-platforms.adoc

```diff
@@ -8,7 +8,7 @@ layout: section
 :sectanchors:
 :toc: left
 
-= Installing Jenkins
+= Installing Jenkins on Different Platforms
```

Comment:
Why? I don't want to install Jenkins on different platforms. I just want to install it on one. Mine.

(Not actually trolling, I think, even though this might look like it.)

Reply:
Hi @daniel-beck and @bitwiseman ,

My rationale for taking @bitwiseman 's original "Installing Jenkins" page (as a _section_ page within the "Getting Started with Jenkins" _chapter_ at https://jenkins.io/doc/book/getting-started/installing/) and moving this page into its own _chapter_ called "Installing Jenkins" is documented in https://issues.jenkins-ci.org/browse/WEBSITE-400.

As someone who's still relatively new to Jenkins and its documentation (like me), it would be easier to find everything that relates to installing Jenkins in a single _chapter_ (rather than this content being fragmented across the handbook). Currently, everything documented about "Installing Jenkins" is nested within a "Getting Started with Jenkins" _chapter_ (which is fair enough), but then there's currently an empty "Advanced Jenkins Installation" _section_ page nested within the "Appendix" _chapter_. Assuming someone wants to add something to the "Advanced Jenkins Installation" page at some point, then for the reader, it would be easier to find everything that relates to installing Jenkins in a single _chapter_ - especially since the jenkins.io documentation currently has no page search feature.

My initial intention was to have a _chapter_ page called "Installing Jenkins" and within that _chapter_, have separate _section_ pages for "Installing Jenkins on Unix/Linux", "Installing Jenkins on Mac OS X", etc. This would entail breaking up the existing page's content into their appropriate pages. (The common bits of information between each platform - like the 'Pre-Install' portion on this page - could be placed in the top "Installing Jenkins" _chapter_ page.) However, I think all of this work (which I suspect would also cover https://issues.jenkins-ci.org/browse/WEBSITE-127) is outside the scope of this pull request.

Nevertheless, to start off this work, I initially decided to nest the existing "Installing Jenkins" _section_ page (currently within the "Getting Started with Jenkins" _chapter_) into a new "Installing Jenkins" _chapter_ and rename this _section_ page to "Installing Jenkins on Different Platforms". This was to avoid having an "Installing Jenkins" _chapter_ page with an "Installing Jenkins" _section_ page (i.e. the page with the existing content on it).

However, in retrospect, I now think this is overkill and is most likely confusing the matter. Therefore, I've now updated this pull request to effectively move the existing "Installing Jenkins" _section_ page into the "Installing Jenkins" _chapter_ page and have the empty "Advanced Jenkins Installation" page nested within this chapter.

At some point in the future, we can create separate "Installing Jenkins on Unix/Linux", "Installing Jenkins on Mac OS X", etc. pages within this new "Installing Jenkins" _chapter_ and migrate the content from this _chapter_ page into their appropriate _section_ pages.

I hope all this makes sense?

Reply:
Incidentally, I've placed in a redirect so that https://jenkins.io/doc/book/getting-started/installing/ will redirect to https://jenkins.io/doc/book/installing/ (i.e. the new _chapter_ page).